### DISCUSSION TITLE
Audible feedback

### DISCUSSION THREAD

**Discussion by innovation-plantation** [VREMSoftwareDevelopment/WiFiAnalyzer#348] [Ideas]
What would be the best way to give audible feedback? The scenario would be you're adjusting things to see what gives the best signal from maybe a wireless router another room that generally gets poor reception, and if you could hear some beeping or something while making adjustments, I think it would really make it simple. Just adding a beep whose volume indicates how strong the signal is whenever it gets scanned would make it infinitely easier to concentrate on the adjustments and get a good position (or maybe trying some supposedly great antenna that might need to be aimed, etc.)

I'm thinking that would be easy to add right away, and then start planning how to incorporate multiple phones so that you could tell if it reaches grandma's bedroom and the living room both from the closet, for example, so in this scenario you'd have some additional phones running the app communicate with your primary phone that's close enough that you can hear it. Somehow get the status of each of the other phones and beep loud or soft according to whether you have a strong signal. Perhaps the different receivers would correspond to different higher or lower pitched tones that the monitoring station would play either together as a musical chord or one after the other like a musical scale.

Or maybe someone has other ideas that would make it easier to implement or how to work it into the UI.

**Comment by VREMSoftwareDevelopment** [comment]
This is a feature request, not a discussion
